app/views/pages/support.html.erb

```diff
@@ -1,7 +1,46 @@
-<% content_for :page_title do %>
-  Support
-<% end %>
+<div class="grid-row inner">
+  <div class="column-two-thirds">
+    <h1 class='heading-large'>Support</h1>
 
-<h1 class="heading-large">
-  Support
-</h1>
+      <p>
+        The Government Digital Service (GDS) maintains the service behind data.gov.uk and helps
+        publishers to manage their data and keep it up-to-date. GDS provides operational support
+        from 9am to 5pm, Monday to Friday.
+      </p>
+
+      <p>
+        If you want to ask for a dataset, choose the data request option.
+      </p>
+
+      <p>
+        For questions about a dataset, you can contact the publisher directly if they’ve provided contact details on the dataset page.
+      </p>
+      <p>
+        To get answers about <%= link_to "GOV.UK", "https://www.gov.uk/" %> you can check the GOV.UK help pages or <%= link_to "contact", "https://www.gov.uk/contact/govuk" %> the team.
+      </p>
+      
+      <form action="tickets/new" method="get">
```

Comment:
This form is missing a csrf token. I think there's an automatic way to do that in rails.